src/modules/ocaml.rs

```diff
@@ -29,6 +30,18 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 _ => None,
             })
             .map(|variable| match variable {
+                "switch" => {
+                    let stdout = context.exec_cmd("opam", &["switch", "-s", "show"])?.stdout;
+                    let switch_raw = stdout.trim();
+                    let path = Path::new(switch_raw);
+                    let normalized = if path.has_root() {
+                        // use named local switch, oft prefixed with full path by opam
```

Comment:
Don't abbreviate comments 🙃
```suggestion
                        // use named local switch, often prefixed with full path by opam
```

Reply:
oh, oft is a word :) uncommon, but not abbreviated :)